catalog/spatial/registry/registry-common/src/main/java/org/codice/ddf/registry/common/metacard/RegistryUtility.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package org.codice.ddf.registry.common.metacard;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.codice.ddf.registry.common.RegistryConstants;
+
+import ddf.catalog.data.Attribute;
+import ddf.catalog.data.Metacard;
+
+public class RegistryUtility {
+
+    /**
+     * Checks that the metacard passed in has the Registry metacard tag and a valid RegistryId.
+     *
+     * @param metacard - will have tags and RegistryId evaluated
+     * @return true if registryId is present in the metacard with the Registry_Tag, false otherwise
+     */
+    public static boolean hasRegistryId(Metacard metacard) {
```

Comment:
Should hasRegistryId check the tags? seems like the tag check should be a separate method.